File: Python/nastyhacks.py
n = int(input())
lstr = []
if 1 <= n <= 100:
    for i in range(n):
        inp = input()
        lst = inp.split()
        if -(10**6) <= int(lst[0]) <= 10**6 and -(10**6) <= int(lst[1]) <= 10**6 and 0 <= int(lst[2]) <= 10**6:

            if int(lst[1]) - int(lst[2]) > int(lst[0]):
                lstr.append("advertise")
            elif int(lst[1]) - int(lst[2]) == int(lst[0]):
                lstr.append("does not matter")
            elif int(lst[1]) - int(lst[2]) < int(lst[0]):
                lstr.append("do not advertise")

for j in lstr:
    print(j)

File: Cpp/nastyhacks.cpp
#include<iostream>
#include<math.h>
using namespace std;

int main(){
    int n;
    cin >> n;
    if (1 <= n && n <= 100){
        for (int i = 0; i < n ; i++){
            int a, b, c;
            cin >> a >> b >> c;
            if (-(pow(10,6)) <= a && a <= pow(10,6) && -(pow(10,6)) <= b && b <= pow(10,6) && -(pow(10,6)) <= c && c <= pow(10,6)){
                if(b - c > a) cout << "advertise" << endl;
                else if(b - c == a) cout << "does not matter" << endl;
                else if(b - c < a) cout << "do not advertise" << endl;
            }   
        }
    }
}
